wc-improved-external-products.php

```diff
@@ -157,13 +158,13 @@ function iepp_external_add_to_cart() {
 		<?php do_action( 'woocommerce_before_add_to_cart_button' ); ?>
 
 		<?php
-		$options = get_option('improvedexternalproducts');
-		if(!empty($options['custom_single_button_html'])){
-			$html = $options['custom_single_button_html'];
-			$html = str_replace('{product_url}', esc_url( $product_url ), $html);
-			$html = str_replace('{target}', $target, $html);
-			$html = str_replace('{button_text}', esc_html( $button_text ), $html);
-			$html = str_replace('{price_html}', $price_html, $html);
+		$options = get_option('woocommerce-improved-external-products');
```

Comment:
@alexmigf This implies that before it was buggy, fetching the option from the wrong place? So apparently nobody is using the feature, if nobody noticed?

Reply:
After discussing this with Ewout, which did this code 4 years ago, it seems a long time bug that no one haven't discovered to date. Because the Pro version uses the removed key, maybe some confusion happened at the time.